client/src/components/modal/modal.tsx

```diff
@@ -9,13 +9,16 @@ interface IModalProps {
  toggleModal(): void;
  socket: Socket;
  userId: string;
+ members: string[];
```

Comment:
I think members isn't used in this component, so maybe can remove from the interface.

Reply:
Yes you are right. I have removed from component